pkg/audit/manager.go

```diff
@@ -482,6 +496,26 @@ func (am *Manager) auditFromCache(ctx context.Context) ([]Result, []error) {
 	return results, errs
 }
 
+// nsMapFromObjs creates a mapping of namespaceName -> corev1.Namespace for
+// every Namespace in input `objs`.
+func nsMapFromObjs(objs []unstructured.Unstructured) (map[string]*corev1.Namespace, error) {
+	nsMap := make(map[string]*corev1.Namespace)
+	for _, obj := range objs {
+		if obj.GetKind() != "Namespace" {
+			continue
+		}
+
+		var ns corev1.Namespace
+		err := runtime.DefaultUnstructuredConverter.FromUnstructured(obj.Object, &ns)
```

Comment:
Thanks for the PR! 
One question: since the original issue mentioned they are syncing namespaces, shouldn’t that data already exist in the OPA cache such that we don’t need to ask the api server for them? We also mention the need to sync namespace here: https://open-policy-agent.github.io/gatekeeper/website/docs//gatekeeper/website/docs/sync/

Reply:
I might be confused about what cache you are referring to by "OPA Cache", but I think the namespaces being processed in the func are from the audit cache. The input to this func is generated from [`auditCache.ListObjects`](https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/audit/manager.go#L457).